ui-tests/cypress/integration/content/management.spec.js

```diff
@@ -1,9 +1,16 @@
 import HomePage from "../../support/pageObjects/HomePage.js";
+import { generateRandomContentTypeCode, generateRandomId } from "../../support/utils.js";
```

Comment:
I moved `generateRandomContentTypeCode` to 'types.spec.js' as it wasn't used anywhere else; move it back to 'utils.js' if you need it before merging

Reply:
Sorry, but I can't seem to see that it has been moved to `types.spec.js` as you mentioned. Are you about to push those changes?

Reply:
Please rebase on develop, it has just been merged "https://github.com/entando/entando-test-automation/blob/develop/ui-tests/cypress/integration/content/types.spec.js"